### PR TITLE
[NO-JIRA] Only use Github and GitLab for submissions

### PR DESCRIPTION
**Pull request by hakimAtPaidy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ We understand that your time is valuable, and in anyone's busy schedule solving
...
 
 ## How to submit?
-You can do this however you see fit - you can email us a tarball, a pointer to download your code from somewhere or just a link to a source control repository. Make sure your submission includes a small **README**, documenting any assumptions, simplifications and/or choices you made, as well as a short description of how to run the code and/or tests. Finally, to help us review your code, please split your commit history in sensible chunks (at least separate the initial provided code from your personal additions).
+Please upload your solution to a version control system hosting service, preferably [Github](https://www.github.com) or [Gitlab](https://www.gitlab.com), that allows code viewing and checkout without requiring an account. Afterward, email us a link to the branch that contains your solution. Make sure your submission includes a small **README**, documenting any assumptions, simplifications and/or choices you made, as well as a short description of how to run the code and/or tests. Finally, to help us review your code, please split your commit history in sensible chunks (at least separate the initial provided code from your personal additions).
```
Comment (Anton-Semenov-Paidy) [inline README.md:14]: Should we only clarify that we will only review the branch sent to us and are unlikely to review other branches?

**Review by Gregor-Robertson-Paidy — COMMENTED** [review]
Can you please update the [submission](https://github.com/paidy/interview/blob/master/SimpleRestaurantApi.md#submission) section of the simple restaurant challenge as well? 🙇

File: README.md (hunk 1 above)
Comment (hakimAtPaidy) [reply README.md:14]: I think there is no reason to be verbose, we only faced this issue once.

**Review by Ali-Imran-Paidy — APPROVED** [review]

**Review by Xeodou-Li-Paidy — APPROVED** [review]

**Review by Anton-Semenov-Paidy — APPROVED** [review]

**Review by Gregor-Robertson-Paidy — APPROVED** [review]
LGTM

**Pull request merged**